src/http/local_async_connection_base.ipp

```diff
@@ -1,6 +1,10 @@
 #ifndef BOOST_NETWORK_PROTOCOL_HTTP_IMPL_LOCAL_ASYNC_CONNECTION_BASE_IPP_20170307
 #define BOOST_NETWORK_PROTOCOL_HTTP_IMPL_LOCAL_ASYNC_CONNECTION_BASE_IPP_20170307
 
+// Note: This file is mostly a copy of
+// boost/network/protocol/http/client/connection/async_base.hpp,
```

Comment:
hpp -> ipp

Reply:
Nope, hpp is correct. The upstream library only has a single file. For various linker-related reasons, we had to split out the specialized function into its own implementation file.